packages-exp/auth-compat-exp/src/phone_auth_provider.ts

```diff
@@ -49,15 +49,15 @@ export class PhoneAuthProvider
       | compat.PhoneMultiFactorSignInInfoOptions,
     applicationVerifier: compat.ApplicationVerifier
   ): Promise<string> {
-    return this.phoneProvider.verifyPhoneNumber(
+    return this._delegate.verifyPhoneNumber(
       // The implementation matches but the types are subtly incompatible
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       phoneInfoOptions as any,
-      unwrap(applicationVerifier)
+      applicationVerifier
```

Comment:
Isn't this a type error since we haven't updated the signatures yet?

Reply:
I didn't get any error. I think they might be duck type compatible.